lib/sen_tweet_web/controllers/stream_controller.ex

```diff
@@ -9,8 +9,14 @@ defmodule SenTweetWeb.StreamController do
     render(conn, "new.html")
   end
 
-  def create(conn, %{"user" => user, "track" => track, "filter_level" => filter_level}) do
-    Bitfeels.twitter_stream(user, track, filter_level)
-    redirect(conn, to: "/admin/streams")
+  def create(conn, %{"track" => track, "filter_level" => filter_level}) do
+    Bitfeels.twitter_stream(conn.assigns[:current_user], track, filter_level)
```

Comment:
Can I do a match on `conn.assigns` to get `user` without modifying `conn`?

Reply:
Yup, `%Plug.Conn{assigns: %{current_user: user}} = conn` should do it.

Pattern matching doesn't modify the underlying structure.